cogs/Misc.py

```diff
@@ -187,9 +206,68 @@ async def edit(text: str) -> None:
                 file=file,
                 ephemeral=True,
             )
-            await edit(f"{LOADING_EMOJI} Dumped {i + 1}/{len(tables)} table(s).")
+            await edit(f"{LOADING_EMOJI} Presented {i + 1}/{len(tables)} table(s).")
+
+        await edit(f"{COMPLETED_EMOJI} Presented {len(tables)} table(s).")
 
-        await edit(f"{COMPLETED_EMOJI} Dumped {len(tables)} table(s).")
+    @VIEW_LOGS_PERMISSIONS
+    @commands.slash_command(
+        name="view-logs",
+        description="View the most recent logs generated by the bot.",
+        guild_ids=None if VIEW_DB_GUILD is None else [VIEW_DB_GUILD],
+        guild_only=VIEW_DB_GUILD is not None,
+    )
+    @option(
+        "file",
+        str,
+        parameter_name="filename",
+        description="The log file you want to view.",
+        required=True,
+        default="log.txt",
+        choices=[
+            OptionChoice(f"{filename} - {description}", filename)
+            for filename, description in LOG_FILES.items()
+        ],
+    )
+    @option(
+        "lines",
+        int,
+        parameter_name="line_count",
+        description="The number of lines to view.",
+        required=True,
+        default=20,
+    )
+    @limit(1)
+    async def view_logs(
+        self,
+        ctx: ApplicationContext,
+        filename: str,
+        line_count: int,
+    ) -> None:
+        if filename not in LOG_FILES:
+            # Discord guarantees that this is impossible, but the pycord library might have issues!
+            # The risk is low, but the consequences would be huge (reading the .env file).
```

Comment:
Could you elaborate on these two lines of comment?

Reply:
I added the comment because future readers might wonder "why is this if-statement here?" since, strictly speaking, the check is not required (the discord API guarantees that it will never happen). But the if-statement's purpose is relatively obvious, so the comment actually raises more questions than it answers. I have removed the comment.